### ISSUE TITLE
Clicking a dangling link makes graph view to become empty

### ISSUE THREAD

**Issue by Taitava** [graydon/obsidian-dangling-links#13]
Hi,

I'm not sure if this issue is caused by this plugin, or the graph view core plugin itself. I have a pinned panel that contains a global graph view. And then I have another panel that is not pinned, so it can freely open notes.

I have the danling links list panel open. From there I look up for a link that I want to create a file for. I click the link to open a file that contains that link. The file opens in the unpinned panel. But at the same time, the graph view panel becomes blank:

![kuva](https://user-images.githubusercontent.com/13002142/127455209-82ea3c27-3f73-4b6b-9667-1ada7a3b5470.png)

**Comment by claremacrae** [comment]
Hi @Taitava, @kosmobious and @mrfonado,

Could you confirm if this is still a problem for you with the latest Obsidian?

**Comment by claremacrae** [comment]
@kosmobius sorry for mis-typing your username above...

> Could you confirm if this is still a problem for you with the latest Obsidian?

**Comment by Taitava** [comment]
Yes, this still occurs with Obsidian 1.5.4 and Dangling links 0.3.0 in sandbox vault with no other community plugins installed.

**Comment by claremacrae** [comment]
Thank you for the replies. I'm removing the `question` label.
